### ISSUE TITLE
IfStatementAssignment error message have switched lines and columns. 

### ISSUE THREAD

**Issue by pawelkg** [phpmd/phpmd#683]
- PHPMD version: 2.7.0
- PHP Version: 7.3.6
- Installation type: composer
- Operating System / Distribution & Version: Windows 10

### Current Behavior
I get errors:
C:\Projects\sitemap\src\Helpers\Url.php:31      Avoid assigning values to variables in if clauses and the like (line '17', column '90').
C:\Projects\sitemap\src\Helpers\Url.php:31      Avoid assigning values to variables in if clauses and the like (line '17', column '91').

On this lines (2nd and 3rd line):
```php
array_walk($query, function (&$val, &$var) {
            $var = rawurlencode($var);
            $val = rawurlencode($val);
        });
```

### Expected Behavior
If this line should be error indeed then line and column are switched. Same behavior when output is html or xml. Other thing is if this should produce this error? I'm think not, but not sure if this is intended to work like this.

**Comment by kylekatarnls** [comment]
Hi @pawelkg, it will be fixed in 2.7.1 thanks to @Jir4:
https://github.com/phpmd/phpmd/pull/657/files

**Comment by pawelkg** [comment]
Great. Please tell me also should this code produce such a message?

**Comment by kylekatarnls** [comment]
Sorry, no it should not throw it, that is an other already fixed bug: #661
